<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214)</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(214, 214, 214)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>XYZ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="plot1_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="PlotWidget" name="plot1" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="plot1_text">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>X: plot1_value_x
Y: plot1_value_y
Z: plot1_value_z</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="plot2_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="PlotWidget" name="plot2" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="plot2_text">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>X: plot1_value_x
Y: plot1_value_y
Z: plot1_value_z</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Bars</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QFrame" name="temp1_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" rowspan="2">
            <widget class="QProgressBar" name="temp1">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #000000</string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="temp1_lbl_h">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>64</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>Temperature1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="temp1_lbl_b">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>+0 °C</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="temp2_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" rowspan="2">
            <widget class="QProgressBar" name="temp2">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #000000</string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="temp2_lbl_h">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>64</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>Temperature2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="temp2_lbl_b">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>+0 °C</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="press_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" rowspan="2">
            <widget class="QProgressBar" name="press">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #000000</string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="press_lbl_h">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>64</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>Pressure</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="press_lbl_b">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>0 Па</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="height_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" rowspan="2">
            <widget class="QProgressBar" name="height">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #000000</string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="height_lbl_h">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>64</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="height_lbl_b">
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
             </property>
             <property name="text">
              <string>0 м</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>DataLogs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextBrowser" name="logs"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LargeButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>650</height>
       </size>
      </property>
      <property name="text">
       <string>haha lois i am large button</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
